inc/Engine/Admin/Settings/Page.php

```diff
@@ -878,20 +879,14 @@ private function assets_section() {
 	private function media_section() {
 		$lazyload_beacon  = $this->beacon->get_suggest( 'lazyload' );
 		$exclude_lazyload = $this->beacon->get_suggest( 'exclude_lazyload' );
-		$webp_beacon      = $this->beacon->get_suggest( 'webp' );
-		$dimensions       = $this->beacon->get_suggest( 'image_dimensions' );
-
-		if ( rocket_valid_key() && ! \Imagify_Partner::has_imagify_api_key() ) {
-			$imagify_link = '<a href="#imagify">';
-		} else {
-			$imagify_link = '<a href="https://wordpress.org/plugins/imagify/" target="_blank" rel="noopener noreferrer">';
-		}
+		/*$webp_beacon      = $this->beacon->get_suggest( 'webp' );*/
```

Comment:
You can completely delete the line instead of commenting it

Reply:
Done